app/controllers/tweets_controller.rb

```diff
@@ -29,6 +30,24 @@ def create
     end
   end
 
+  # GET /tweets/:id/edit
+  def edit
+    @tweet = Tweet.find(params[:id])
```

Comment:
ログインしているユーザー以外のものが見えたらダメなので、以下のようにしよう。
```suggestion
    @tweet = current_user.tweets.find(params[:id])
```